airtonix/bankdownloader
# feat: switch config to viper and support gopass

# Description

- [x] config file is now just `json` 
- credential support
  - [x] from the config file
  - [x] from gopass
  - [x] from gopass with a totp
  - [x] from libsecrets
  - ~~[ ] from libsecrets with a totp (not sure if this is possible yet)~~

## What kind of change is this?


## Steps to test changes



1. Build the app with `command to build`
2. Run the app with `./bank-downloader --required-args`
3. Configure the app ...
4. Do this in the app ...
5. Then this ...
6. Observe this in the app ... to confirm changes.

---

> 🎧  [Raw Language, Compassion (Forest Swords)](https://music.youtube.com/watch?v=3oGiYXOuxFs)
